### ISSUE TITLE
Errors in TutorialKriging.m

### ISSUE THREAD

**Issue by Adolphus8** [cossan-working-group/OpenCossan#4]
• Error using KrigingModel (line 46) (DACE toolbox not installed)
• Error in TutorialKriging (line 62)

**Comment by tokamaster** [comment]
Hi @Adolphus8,

For the first error you have to get the ooDACE toolbox developed at Ghent University. You can find it here: http://www.sumo.intec.ugent.be/ooDACE
Once you have downloaded it, move its content into OpenSourceSoftware/src/dace/.

Can you please give me more information about the second error?

**Comment by tokamaster** [comment]
I have run it importing the ooDACE toolbox and there is a conflict between two classes (Optimizer), since OpenCossan and ooDACE have their own Optimizer. To solve this you have to include the ooDACE toolbox into the OpenSourceSoftware/src/dace/ folder so OpenCossan can detect you have it; but also you need to add to path an external ooDACE folder, so the classes don't conflict with each other.

```matlab
>> TutorialKriging
calling lhsdesign
* Nsamples: 5
* Nrv: 2
* iteration: 5
* criterion: none
* smooth: on
[Status:Evaluator  ]       * Processing solver 1/1
===================================================================
 KrigingModel Object  -  Description: Kriging of tip displacement of cantilever beam
===================================================================
 * Type of KrigingModel: regpoly0
 * Reference Model has been defined
 * Inputs  : "XP" "Xh" 
 * Outputs : "disp" 
 * Input data for calibration has been defined
 * Output data for calibration has been defined
 * Metamodel calibrated: regression error R^2: 1
[Status:Evaluator  ]       * Processing solver 1/1
===================================================================
 KrigingModel Object  -  Description: Kriging of tip displacement of cantilever beam
===================================================================
 * Type of KrigingModel: regpoly0
 * Reference Model has been defined
 * Inputs  : "XP" "Xh" 
 * Outputs : "disp" 
 * Input data for calibration has been defined
 * Output data for calibration has been defined
* Input data for validating has been defined
* Output data for validating has been defined
 * Metamodel calibrated: regression error R^2: 1
 * Metamodel validated: regression error R^2: 0.7593
[Status:Evaluator  ]       * Processing solver 1/1
[Simulation:exportResults] Writing partial results (SimulationData_batch_1_of_1) on the folder: /home/enmidol/stable/tmp/20190709T121531
Maximum no. of samples reached. Samples computed 1000; Maximum allowed samples: 1000
===================================================================
FailureProbability Object  -  Description: 
===================================================================
* Results obtained with MonteCarlo method
** First Moment
*** Pfhat     = 6.000e-03
*** Std       = 2.443e-03
*** CoV       = 4.072e-01
** Second Moment
*** variance  = 5.970e-03
** Simulation details
*** # samples  = 1.000e+03
*** # batches  =         1
*** # lines    =         0
*** Exit Flag = Maximum no. of samples reached. Samples computed 1000; Maximum allowed samples: 1000
[Simulation:exportResults] Writing partial results (SimulationData_batch_1_of_1) on the folder: /home/enmidol/stable/tmp/20190709T121531
Maximum no. of samples reached. Samples computed 1000; Maximum allowed samples: 1000
===================================================================
FailureProbability Object  -  Description: 
===================================================================
* Results obtained with MonteCarlo method
** First Moment
*** Pfhat     = 0.000e+00
*** Std       = 0.000e+00
*** CoV       =       NaN
** Second Moment
*** variance  = 0.000e+00
** Simulation details
*** # samples  = 1.000e+03
*** # batches  =         1
*** # lines    =         0
*** Exit Flag = Maximum no. of samples reached. Samples computed 1000; Maximum allowed samples: 1000
```

**Comment by JCSadeghi** [comment]
fixed by [0b120ad ](https://github.com/cossan-working-group/OpenCossan/commit/0b120ad9644e10cdca40ae7ab5b7a906bc5cb8da)
